waves-grpc/src/main/protobuf/waves_blockchain_api.proto

```diff
@@ -26,17 +26,14 @@ service WavesBlockchainApi {
 
     rpc RunAddressScript (RunAddressScriptRequest) returns (RunScriptResponse);
 
-    // TODO remove after release 2.1.3
-    rpc SpendableAssetBalance (SpendableAssetBalanceRequest) returns (SpendableAssetBalanceResponse);
-
     rpc SpendableAssetsBalances (SpendableAssetsBalancesRequest) returns (SpendableAssetsBalancesResponse);
 
     rpc ForgedOrder (ForgedOrderRequest) returns (ForgedOrderResponse);
 
-    // TODO remove after release 2.1.2
-    rpc GetBalanceChanges (google.protobuf.Empty) returns (stream BalanceChangesResponse);
-
-    // TODO rename to GetBalanceChanges after release 2.1.2
+    // TODO after release 2.1.2:
+    // 1. Add GetBalanceChanges with the same functionality
```

Comment:
This is 2.2.0 branch, why not to delete?

Reply:
We can't, because during update both servers won't see this method